README.adoc

```diff
@@ -0,0 +1,88 @@
+# HeuDiConv
+
+[%autowidth, cols="1,1,1", frame=none, grid=none]
+|===
+a|
+image::https://img.shields.io/docker/v/fnndsc/pl-heudiconv?sort=semver[https://hub.docker.com/r/fnndsc/pl-heudiconv]
```

Comment:
```suggestion

```

I think this one can be removed for now, I don't think there's an equivalent for quay.io yet.